README.md

```diff
@@ -108,17 +108,16 @@ to create an AOD instance for each.
     -   Set up
         [CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)
         with the group to approve AOD requests
-    -   Branch protection on the `main` (default) branch
+    -   Add a new branch Ruleset on the `main` (default) branch
+        -   Require signed commits
+        -   Restrict updates
```

Comment:
Made an assumption here that this is the equivalent to `lock branch`. 

Based off of https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/available-rules-for-rulesets#restrict-updates